content/docs/beyond-the-basics/compliance-policies/index.md

```diff
@@ -4,11 +4,371 @@
+---
+
+## Creating a Compliance Policy
+
+Compliance policies are user-defined rule sets that Assets can be tested against. Compliance policies only need to be created once; all applicable Assets will be tested against that policy thereafter. 
+
+For example, a policy might assert that “Maintenance Alarm Events must be answered with a Maintenance Report Event, recorded within 72 hours of the alarm”. This creates a Compliance Policy in the system which any Asset can be tested against as needed.
+
+RKVST allows for several types of Compliance Policies: 
+
+1. ***COMPLIANCE_SINCE:*** checks the time elapsed since a specific type of Event has not exceeded set threshold. 
+
+For example, "time since last maintenance must be less than 72 hours".
+
+{{< tabs name="compliance_since" >}}
+{{{< tab name="UI" >}}
+{{< img src="ComplianceSinceAsset.png" alt="Rectangle" caption="<em>Add Asset Information</em>" class="border-0" >}}
+{{< img src="ComplianceSinceCompliance.png" alt="Rectangle" caption="<em>Add Compliance Information</em>" class="border-0" >}}
+
+{{< /tab >}}
+{{< tab name="YAML" >}}
+```yaml
+---
+steps:
+  - step:
+      action: COMPLIANCE_POLICIES_CREATE
+      description: Create COMPLIANCE_SINCE policy
+      print_response: true
+    description: Maintenance should be performed every 72h
+    display_name: Regular Maintenance
+    compliance_type: COMPLIANCE_SINCE
+    asset_filter:
+      - or: [ "attributes.arc_home_location_identity=locations/<location-id>" ]
+    event_display_type: Maintenance Performed
+    time_period_seconds: "259200"
+```
+Use the [archivist_runner](https://python.rkvst.com/runner/index.html) command to run your YAML file!
+ 
+```bash
+$ archivist_runner \
+      -u https://app.rkvst.io \
+      --client-id <your-client-id> \
+      --client-secret <your-client-secret> \
+      <path-to-yaml-file>
+```
+{{< /tab >}}
+{{< tab name="JSON" >}}
+```json
+{
+    "compliance_type": "COMPLIANCE_SINCE",
+    "description": "Maintenance should be performed every 72h",
+    "display_name": "Regular Maintenance",
+    "asset_filter": [
+        { "or": ["attributes.arc_home_location_identity=locations/<location-id>"]}
+    ],
+    "event_display_type": "Maintenance Performed",
+    "time_period_seconds": "259200"
+}
+```
+Use the curl command to run your JSON file! See instructions for [creating your `BEARER_TOKEN_FILE`](https://docs.rkvst.com/docs/rkvst-basics/getting-access-tokens-using-app-registrations/) here.
+ 
+```bash
+curl -v -X POST \
+    -H "@$BEARER_TOKEN_FILE" \
+    -H "Content-type: application/json" \
+    -d "@/path/to/jsonfile" \
+    https://app.rkvst.io/archivist/v1/compliance_policies
+```
+{{< /tab >}}}
+{{< /tabs >}}
+
+2. ***COMPLIANCE_CURRENT_OUTSTANDING:*** checks if there is an answering Event addressing an outstanding Event.
+
+To correlate Events, define the attribute `arc_correlation_value` in the Event Attributes and set it to the same value on each pair of events that are to be associated.
```

Comment:
Add a note that it is important to ensure the arc_correllation_value is unique to each pair of events so that they are properly tracked and assessed.

Reply:
Notes added